diff --git a/lib/src/Http2Transport.cc b/lib/src/Http2Transport.cc
index f3f153e..11492b3 100644
--- a/lib/src/Http2Transport.cc
+++ b/lib/src/Http2Transport.cc
@@ -996,9 +996,8 @@ void Http2Transport::onRecvMessage(const trantor::TcpConnectionPtr &,
     {
         reconnectionIssued = true;
         LOG_TRACE << "Reconnect due to running out of stream ids.";
-        connPtr->getLoop()->queueInLoop([connPtr = this->connPtr]() {
-            connPtr->shutdown();
-        });
+        connPtr->getLoop()->queueInLoop(
+            [connPtr = this->connPtr]() { connPtr->shutdown(); });
     }
 }